extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -65,10 +68,19 @@ class AmpLightbox extends AMP.BaseElement {
       right: 0,
     });
 
+    if (this.isScrollable_) {
+      st.setStyles(this.element, {
+        overflowY: 'auto',
+        overflowX: 'hidden',
```

Comment:
The `i-amp-html-wrapper` that is applied to `html` already have `-webkit-overflow-scrolling: touch;`